server/src/config.ts

```diff
@@ -253,7 +253,7 @@ export const defaults = Object.freeze<SystemConfig>({
     buttonText: 'Login with OAuth',
     clientId: '',
     clientSecret: '',
-    defaultStorageQuota: 0,
+    defaultStorageQuota: undefined,
```

Comment:
Semantically I would probably prefer `null` here instead of `undefined`, since from my perspective we explicitly want to say "should be unlimited" and not "we don't define the value because we don't care about it". Would be interested to hear @jrasm91 thoughts on this though.